src/styles/index.scss

```
@@ -1,3 +1,138 @@
+$button-color:#00acdc;
+$second-text:#616070;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background: rgb(213, 213, 209);
+}
+
+$star-size: 16px;
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: $star-size;
+    height: $star-size;
+    flex-shrink: 0;
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+    background-size: auto;
+
+    &:not(:last-child) {
+      margin-right: $star-size / 4;
+    }
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
+
+.product-card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  position: relative;
+  background: #fff;
+  margin: 0;
+  color: #060b35;
+
+  &__img {
+    display: block;
+    padding: 32px 19px 0;
+
+    &-size {
+      display: block;
+      max-width: 100%;
+      height: 134px;
+    }
+  }
+
+  &__body {
+    display: flex;
+    flex-direction: column;
+    padding: 0 16px 16px;
+  }
+
+  &__description {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin: 0;
+    padding-top: 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    padding-top: 4px;
+    color: $second-text;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 16px;
+    align-items: center;
+    text-align: center;
+
+    &-reviews {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 24px;
+    align-items: center;
+
+    &-value {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $second-text;
+    }
+
+    &-cost {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    background-color: $button-color;
+    height: 40px;
+    padding: 12px 70px;
```

you can do like this
```suggestion
    padding: 12px;
    text-align: center;
```